src/coreclr/clrfeatures.cmake

```diff
@@ -39,3 +39,7 @@ endif()
 if (CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_MACCATALYST OR CLR_CMAKE_TARGET_IOS OR CLR_CMAKE_TARGET_TVOS)
   set(FEATURE_OBJCMARSHAL 1)
 endif()
+
+if (CLR_CMAKE_TARGET_UNIX)
```

Comment:
Let's move this up to line 35 and merge with the Win32 version.

`if (CLR_CMAKE_TARGET_WIN32 OR CLR_CMAKE_TARGET_UNIX)`

Reply:
ok